package.json

```diff
@@ -8,8 +8,16 @@
   "type": "module",
   "exports": {
     ".": {
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.cjs"
+      "import": {
+        "types": "./dist/index.d.mts",
```

Comment:
I think once `unbuild` is updated in this repo then this explicit `types` line will not be required.